examples/http-server/main.go

```diff
@@ -60,3 +64,31 @@ func fooHandler(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 	w.Write([]byte(fmt.Sprintf("{\"message\": \"Hello %s\"}", p.Name)))
 }
+
+func outgoingCallHandler(w, http.ResponseWriter, r *http.Request) {
+	client := http.Client{
+		Transport: hyperhttp.NewTransport(
+			http.DefaultTransport,
+		),
+	}
+
+	req, _ := http.NewRequestWithContext(r.Context(), "GET", "https://httpbin.org/get", nil)
```

Comment:
Comment on this line and say that "In order to correlate the server call to this client call we need to pass the server request's context."